backend/backend.go

```diff
@@ -88,6 +88,7 @@ var fixedURLWhitelist = []string{
 	"https://pocketbitcoin.com/",
 	// BTCDirect
 	"https://btcdirect.eu/",
+	"https://checkout.btcdirect.eu/",
```

Comment:
adding here is not strictly needed unless we add an in-app link to this url.

Reply:
Maybe remove it then?

Reply:
The only reason I'd keep it here is so we know this is an url we allow.